src/api/api.service.ts

```diff
@@ -300,4 +303,32 @@ export class ApiService {
       );
     }
   }
+
+  /**
+   * It gets all the APIs and their endpoints from the database
+   * @param {string} profileId - string - the id of the profile
+   * @returns An array of objects.
+   */
+  async getDPD(profileId: string): Promise<any> {
+    try {
+      const apis = (await this.apiRepo.find({ where: { id: profileId } })).map(
```

Comment:
You're trying to get all APIs that belong to a profile but your condition is querying the Id instead of profileId 

For every request that is what I get 
![IMG_20220926_113622_1664189800444](https://user-images.githubusercontent.com/38318744/192259937-a0fbb9fb-9462-4581-ae71-5ad35aacf87b.jpg)